kroxylicious-filters/kroxylicious-authorization/src/main/java/io/kroxylicious/filter/authorization/AuthorizationFilter.java

```diff
@@ -82,6 +82,9 @@ CompletionStage<ResponseFilterResult> onResponse(ResponseHeaderData header, ApiV
         apiEnforcement.put(ApiKeys.FIND_COORDINATOR, new Passthrough<>(0, 6));
         apiEnforcement.put(ApiKeys.JOIN_GROUP, new Passthrough<>(0, 9));
         apiEnforcement.put(ApiKeys.SYNC_GROUP, new Passthrough<>(0, 5));
+        apiEnforcement.put(ApiKeys.DESCRIBE_CONFIGS, new Passthrough<>(1, 4));
+        apiEnforcement.put(ApiKeys.ALTER_CONFIGS, new Passthrough<>(0, 2));
+        apiEnforcement.put(ApiKeys.INCREMENTAL_ALTER_CONFIGS, new Passthrough<>(0, 1));
```

Comment:
A `ConfigResource` can name a topic, and should be subject to authz (see https://docs.confluent.io/platform/current/security/authorization/acls/overview.html#topic-resource-type-operations for example). So I don't think it's as simple as adding Passthrough.

Reply:
done